app/views/users/dashboard.html.erb

```diff
@@ -16,10 +16,9 @@
           <p><%= booking[:payed] ? 'yes' : 'no' %></p>
         </div>
 
-        <div class= "dog-owner">
-        <%= link_to "Go to your dog's page", dog_path(@dogs[i][:id]), class: "links-sessions" %>
-        <%= link_to "Renter's profile", user_path(@renter[i][:id]), class:"btd-yel" %>
-        <%= link_to 'Delete this booking', booking_path(booking), method: :delete,
+        <div class= "book-actions">
+        <%= link_to "#{booking.dog.name.upcase}'S PROFILE", dog_path(@dogs[i][:id]), class:"btd-yel" %><br><br>
```

Comment:
U mean @dogs[i][:name].upcase ?

Reply:
nope, the dog of the booking in particular

Reply:
Ok so it is '@dogs[i][:name].upcase' then. (check my comments in the booking controller to know exactly what @dogs is passing)
You are making a DB request in the view with that code anyway